src/layout/VerifyEmail.component.js

```diff
@@ -9,6 +9,9 @@ const sendEmailVerificationMutation = {
     type: 'create',
 }
 
+const emailRegex =
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/i
```

Comment:
Suggest we use the RegEx that is used for the field validation for consistency and then potentially update both the field/button validator in the context of updating the email update workflow.

If we use different RegExes, the email can potentially fail front-end validation, not be posted to the backend, and then the verify email button will be enabled, but send to the old email address